src/pages/index.astro

```diff
@@ -114,10 +114,11 @@ const recentReleases = releases.slice(0, 3);
 		margin-left: auto;
 		max-height: 10em;
 		object-fit: cover;
-		margin-top: -2em;
+		margin-top: -2em; /* TODO: replace with different grafic - this one here has issues because it draws top lines  */
 		float: right;
 		margin-left: 1.5em;
 		margin-bottom: 1.5em;
+        padding: 10px; 
```

Comment:
We have now decided to remove the titles completely because they are often pretty unnecessary and it is recognizable from the navbar on which page a user is